.gitignore

```diff
@@ -45,7 +45,7 @@ lerna-debug.log*
 
 # IDE - VSCode
 .vscode/*
-!.vscode/settings.json
+.vscode/settings.json
```

Comment:
Should settings not be committed? Can you not override them if this file is committed?

Reply:
Ah yeah, I should've checked with you on that. You can't override settings if this file is committed, which is why I made this update. Unless there's a different way of adding project-level settings that I'm not aware of

Reply:
Mmmm I like this idea, but I hate having to fight shared settings. It's hard for me to know what's best here because I don't use vscode. Which of your settings didn't match, if I may ask?

Reply:
The only setting I updated was this one:
```json
  "editor.formatOnSave": true,
```

I like to auto-format on every save.

Reply:
Ah. we have that on the frontend. Ah you were the one committing it too lol. Would you mind just committing it here too?

I'm not sure about the `defaultFormatter`. If we could invoke eslint instead I think that would be best. It has other logic like removing unused imports, etc.
Maybe it's too heavy though.

Reply:
Word, I updated it! I actually tried to enable eslint formatting at first, but I couldn't get it to work for some reason and kept the default.